workers/unity/Packages/com.improbable.gdk.core/Components/Option.cs

```diff
@@ -2,14 +2,30 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     An implementation of Option which is compatible with Unity's ECS.
+    /// </summary>
+    /// <remarks>
+    ///     This is required because bool is not blittable by default.
```

Comment:
is it only for bools?

or do you mean "nullable"?

Reply:
No as in the bool inside the Option used for `hasValue` needs to be blittable

Reply:
They might?

Reply:
I don't think this belongs in the xml docs. They shouldn't every be using this type, it's just a hacked together thing that we can use for updates.